code/game/machinery/spaceheater.dm

```diff
@@ -143,32 +145,34 @@
 		cell.emp_act(severity)
 
 /obj/machinery/space_heater/attackby(obj/item/I, mob/user, params)
+	. = ..()
```

Comment:
Should we really be doing this? don't the parents of attackby do things like try and hit the machine with the tool?

Reply:
In theory yes, it attacks the object with the item you are wielding, but if you return TRUE the obj won't be hit, is all tested and working as it should

Reply:
I'll hold you to that